packages/hdwallet-native/package.json

```diff
@@ -18,6 +18,7 @@
     "@shapeshiftoss/hdwallet-core": "1.12.1",
     "@terra-money/terra.js": "^1.6.0",
     "bchaddrjs": "^0.4.9",
+    "big.js": "^6.0.3",
```

Comment:
we use `bignumber.js` everywhere else. I don't know the difference but I'd prefer to be consistent.

Reply:
I used `big.js` because it's what the BNB SDK uses to do its math, so it would be guaranteed to be available in any project using the SDK. No problem switching though.